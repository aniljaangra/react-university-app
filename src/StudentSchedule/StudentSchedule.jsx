/**
 * Created by Anil Jangra on 4/22/2021
 */
import React from 'react';
import Grid from '@material-ui/core/Grid';
import AttendanceRecord from './AttendanceRecord/AttendanceRecord.container';
import Scheduler from './Scheduler/Scheduler.container';

const StudentSchedule = () => (
  <Grid container>
    <Grid item sm={5}>
      <AttendanceRecord />
    </Grid>
    <Grid item sm={1} />
    <Grid item sm={6}>
      <Scheduler />
    </Grid>
  </Grid>
);

export default StudentSchedule;
