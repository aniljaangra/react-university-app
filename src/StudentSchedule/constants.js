/**
 * Created by Anil Jangra on 4/22/2021
 */
export const ATTENDANCE = {
  PRESENT: 'present',
  ABSENT: 'absent',
};
export const SUBJECTS = {
  POTIONS_MASTER: 'Potions Master',
  DEFENSE: 'Defense Against the Dark Arts',
};
export const SET_TEACHER_ATTENDANCE = 'SET_TEACHER_ATTENDANCE';
export const NOT_ASSIGNED = 'Not Assigned';
