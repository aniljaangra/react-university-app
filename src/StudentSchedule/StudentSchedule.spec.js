import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import StudentSchedule from './StudentSchedule';

describe('StudentSchedule', () => {
  it('renders correctly', () => {
    const tree = shallow(<StudentSchedule />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
