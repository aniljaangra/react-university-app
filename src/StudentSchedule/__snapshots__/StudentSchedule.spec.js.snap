// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StudentSchedule renders correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={5}
  >
    <AttendanceRecord />
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={1}
  />
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={6}
  >
    <Scheduler />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
